src/avfoundation.cs

```diff
@@ -10934,15 +10924,14 @@ interface AVContentKeyRecipient {
 	[TV (10,2), Mac (10,12,4), iOS (10,3), NoWatch]
 	[BaseType (typeof (NSObject))]
 	interface AVContentKeySession {
-
-		// used with Create (string keySystem, [NullAllowed] NSUrl storageUrl);
-		[TV (10, 2), Mac (10, 12, 4), iOS (10, 3)]
-		[Field ("AVContentKeySystemFairPlayStreaming")]
-		NSString FairPlayStreaming { get; }
-
 		[Static]
+		[Internal]
```

Comment:
Use `[Advanced]` so the current SDK can be used on a newer OS version (or a non-documented) `NSString` constant. That tells the IDE to hide the API from normal intellisense (but it's still available if needed).